### ISSUE TITLE
Input element type checkbox missing accessible name

### ISSUE THREAD

**Issue by Sukha11y** [Sukha11y/Test-ZT-fix#5]
Problem
Where is it located
index.html - [https:///c%3A/Users/SukhadaPawar/Test-ZT-fix/Test-ZT-fix/index.html](https://c/Users/SukhadaPawar/Test-ZT-fix/Test-ZT-fix/index.html)

Locate element
XPath or CSS selector
body>div:nth-of-type(6)>*:nth-child(1)>*:nth-child(1)>*:nth-child(2)>*:nth-child(1)

Copy selector to clipboard
HTML element
<input type="checkbox" id="new">

Copy HTML element to clipboard
Description
One or more input type='checkbox' elements, without an ARIA-assigned role, not intentionally hidden in the DOM and available to assistive technologies, does not have a mechanism that allows an accessible name value to be calculated.
Why it matters
When input elements do not have an accessible name, assistive technologies may incorrectly render the name or provide no name at all to users. When an accessible name is not present, users of assistive technologies may not be able to complete a form.

Solution
What to do
Provide an accessible name for input elements with the label element whenever an on-screen label is present. The label element's value is then explicitly associated with the relevant form field using the for and id attributes. Use of the label element provides an added benefit as it increases the target area that users can click to focus the form field. In certain instances, however, it is impractical or 'undesirable' to use an explicit label for a form field; this is largely driven by the fact that the value between the opening and closing label tags must, by definition, be a block of displayed text. When on-screen text is not available or can not programmatically be contained in a label (such as when a field is placed in a data table with a column label and a row label) - authors can use another method to provide an accessible name such as by using the title, aria-label, or aria-labelledby attribute to provide the label. The aria-labelledby attribute must contain one or more IDs of elements that label the form field. The aria-label attribute should only be used when on-screen labels can not be used. Authors should place the string that labels the form field in the aria-label attribute.
